Source/p_spec.c

```diff
@@ -2505,7 +2506,15 @@ void P_SpawnSpecials (void)
       case 271:   // Regular sky
       case 272:   // Same, only flipped
         for (s = -1; (s = P_FindSectorFromLineTag(lines+i,s)) >= 0;)
+        {
+          // Texture comes from upper texture of reference sidedef
```

Comment:
Does this have to be in the `for()` loop?

Reply:
Well, in the lines above this is separated. Also, not fixing inventory bugs does not mean to introduce similar new ones. 😉

Reply:
Do you mean this: https://github.com/fabiangreffrath/woof/pull/483/commits/a9682012aefb4cbba0c695ccc6358b062cdbdf0f ?

Reply:
Beautiful, thank you!